Book-A-Desk.Api/BookingNotifier.fs

```diff
@@ -1,53 +1,93 @@
 namespace Book_A_Desk.Api
 
+open FsToolkit.ErrorHandling
+open MailKit.Net.Smtp
+open MailKit.Security
+open MimeKit
 open System
 open Book_A_Desk.Api.Models
 open Book_A_Desk.Domain.Office.Domain
 open Book_A_Desk.Domain.QueriesHandler
-open MailKit.Net.Smtp
-open MailKit.Security
-open MimeKit
 
 type BookingNotifier =
     {
-        NotifySuccess: Booking -> Async<bool>
+        NotifySuccess: Booking -> Async<Result<unit, string>>
```

Comment:
Booking Notifier now follows the rest of the program by sending a Result to indicate success/failure instead of a bool